app/testOnly/views/StubAgentClientLookup.scala.html

```diff
@@ -16,11 +16,15 @@
 
 @import helper.CSRF
 @import testOnly.models.StubAgentClientLookupModel
-@import views.html.templates.inputs.text
+@import views.html.templates.inputs.Text
+@import views.html.MainTemplate
+
+@this(mainTemplate: MainTemplate,
+text: Text)
 
 @(form: Form[StubAgentClientLookupModel], redirectUrl: String)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
 
-@views.html.main_template(title = "Stub Agent Client Lookup Frontend", bodyClasses = None, appConfig = appConfig) {
+@mainTemplate(title = "Stub Agent Client Lookup Frontend", bodyClasses = None, appConfig = appConfig) {
 
     @helper.form(action = testOnly.controllers.routes.StubAgentClientLookupController.post) {
```

Comment:
On all your views that use it, could you inject FormWithCSRF instead of doing `@helpers.form` please?